db/models/item.js

```diff
@@ -22,6 +22,12 @@ module.exports = db => db.define('items', {
       isInt: true
     }
   }
+}, {
```

Comment:
great job, jeff! all look good. 👍 
you might want to change price type on line 9 here too?